eventually/authentication/models.py

```diff
@@ -102,32 +134,43 @@ def myattr_to_dict(self):
             'updated_at': int(self.updated_at.timestamp())
             }
 
-    #'FIXME: rewrite docs'
-    def update(self,
-               first_name=False,
-               last_name=False,
-               middle_name=False,
-               email=False,
-               password=False):
+    def update(self, first_name=None, last_name=None, middle_name=None, email=None, password=None):
         """
         Updates user profile in the database with the specified parameters.\n
         :param first_name: <str> User's first name
         :param last_name: <str> User's last name
         :param middle_name: <str> User's middle
         :param email: <str> User's  email
         :param password: <str> User's password
-        :return: returns True if object updated or False if parameters were incorrect
+        :return: returns True if object updated, False if parameters were incorrect
+                 or -1 if none of the conditions succeded.
         """
-        user = CustomUser.get_by_id(self.id)
+        user_changed = False
+        if self.id:
+            user = CustomUser.get_by_id(self.id)
+        else:
+            print("You're trying to modify a user which does not exist in the database")
```

Comment:
`print` ?  can you replace them with `logs` ?

Reply:
Prints removed, logger will be set up additionally a bit later.